whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/client/NrtmRestClient.java

```diff
@@ -82,7 +83,8 @@ public List<String> getNrtmAvailableSources(){
     public String getNotificationFileSignature(final String source){
```

Comment:
Are you reading the signature separately from the notification content?

Reply:
the signature is read from https://nrtm-prepdev.db.ripe.net/nrtmv4/RIPE/update-notification-file.jose

Reply:
and Mahesh suggested to create a resource with the public signature. I think that it is up to the user to do what he want with the signature

Reply:
Public key not public signature. Signature and payload should be in one response from jose url

Reply:
yes, I am getting both from https://nrtm-prepdev.db.ripe.net/nrtmv4/RIPE/update-notification-file.jose
as we already talked last week